### PR TITLE
Auth lang fix

### PR DESCRIPTION
**Pull request by kutay-celebi**

### CODE REVIEW

File: kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/config/ResourceServerConfig.java [hunk 1]
```diff
@@ -44,14 +44,16 @@ public void configure(HttpSecurity http) throws Exception {
             .exceptionHandling()
             .authenticationEntryPoint(entryPoint)
             .accessDeniedHandler(accessDeniedHandler)
-            //.and().cors().configurationSource(corsFilter()).and().csrf().disable(); // todo buralari aktif edelim. Suanlik sorun yok.;
-        .and().cors().disable().csrf().disable(); // todo buralari aktif edelim. Suanlik sorun yok.;
+            // TODO: @kutay-celebi 07.03.2020 will be able
+            //.and().cors().configurationSource(corsFilter()).and().csrf().disable();
```
Comment (azizerel) [inline kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/config/ResourceServerConfig.java]: comment code ?

File: kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/controller/UserRestController.java [hunk 2]
```diff
@@ -28,10 +27,8 @@
     //    return new KuartzResponse<>(oAuth2Authentication.getName());
```
Comment (azizerel) [inline kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/controller/UserRestController.java]: comment?

File: kuartz-authorization/kuartz-auth-client/src/main/java/com/kuartz/core/auth/dto/RoleModel.java [hunk 3]
```diff
@@ -5,7 +5,9 @@
 import javax.validation.constraints.NotNull;
 import java.util.List;
 
+// FIXME: @kutay-celebi 07.03.2020 add constraints
 public class RoleModel extends KuartzModel {
+
     @NotNull
     private String code;
```
Comment (azizerel) [inline kuartz-authorization/kuartz-auth-client/src/main/java/com/kuartz/core/auth/dto/RoleModel.java]: @NotBlank ?

File: kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/config/ResourceServerConfig.java (hunk 1 above)
Comment (kutay-celebi) [reply kuartz-authorization/authz-service/src/main/java/com/kuartz/auth/config/ResourceServerConfig.java]: L:47 todo.

**Pull request merged**